scripts/scripts.js

```diff
@@ -120,12 +120,29 @@ export function formatDate(date) {
   });
 }
 
+/**
+ * Builds an article sidebar and appends it to main in a new section.
+ * @param main
+ */
+async function buildArticleSidebar(main) {
+  if (main.querySelector('.sidebar') || !main.firstChild) {
```

Comment:
I like the check for an existing sidebar, as on some pages it will be in the word document 👍